OsmAnd/src/net/osmand/plus/views/mapwidgets/RouteInfoWidgetsFactory.java

```diff
@@ -1271,6 +1274,15 @@ public boolean updateInfo(DrawSettings drawSettings) {
 						} else {
 							locimgId = R.drawable.warnings_pedestrian;
 						}
+					} else if(alarm.getType() == AlarmInfoType.TUNNEL) {
+						DecimalFormat df = new DecimalFormat("0.#");
+						if(settings.DRIVING_REGION.get().americanSigns){
```

Comment:
Should be in OsmAndFormatter. alarm.getFloatValue() - should be always in meters!